src/JoomlaBrowser.php

```diff
@@ -897,4 +897,34 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+    
+    /**
+     * Function to delete the menu item
+     *
+     * @param   string  $menuItem  Menu Item to be deleted
+     * @param   string  $menu      Menu type of the menu item
+     *
+     * @return void
+     */
+    public function deleteMenuItem($menuItem, $menu)
+    {
+        $I->amOnPage('administrator/index.php?option=com_menus&view=menus');
+        $click(['xpath' => "//a[contains(text()," . $menu . ")]"]);
+        $I->waitForText('Menus: items (' . $menu . ')', '30', ['css' => 'h1']);
+        $I->searchForItem($title);
+        $I->click(['xpath' => "//input[@id='cb0']"]);
+        $I->click(['xpath'=> "//button[@onclick=\"if (document.adminForm.boxchecked.value==0){alert('Please first make a selection from the list.');}else{ Joomla.submitbutton('items.trash')}\"]"]);
```

Comment:
That selector needs to be improved :-) Does $I->clickToolbarButton('Trash'); not work?